blocks/product-information-block/features/product-information/default.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { useFusionContext } from "fusion:context";
+
+import { Heading, HeadingSection, Price, Stack } from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-product-information";
+
+const getPrice = (type, prices) => (prices && prices.filter((x) => x.type === type)?.[0]) || null;
+
+export const ProductInformationDisplay = ({ data }) => {
+	const pricing = (data?.pricing && data?.pricing[0]) || null;
```

Comment:
looks good getting the data 
<img width="712" alt="Screen Shot 2022-07-21 at 06 39 27" src="https://user-images.githubusercontent.com/5950956/180194923-9bf0a08f-ded4-4c11-af19-eeb5e1198b47.png">